tools/ci/azure/install_chrome.yml

```diff
@@ -1,6 +1,9 @@
 steps:
-# This is equivalent to `Homebrew/homebrew-cask-versions/google-chrome-dev`,
-# but the raw URL is used to bypass caching.
-- script: HOMEBREW_NO_AUTO_UPDATE=1 brew cask install https://raw.githubusercontent.com/Homebrew/homebrew-cask-versions/master/Casks/google-chrome-dev.rb
+# The conflicting google-chrome and chromedriver casks are first uninstalled.
+# The raw google-chrome-dev cask URL is used to bypass caching.
+- script: |
+    HOMEBREW_NO_AUTO_UPDATE=1 brew cask uninstall google-chrome
+    HOMEBREW_NO_AUTO_UPDATE=1 brew cask uninstall chromedriver
```

Comment:
If you brew cash uninstall something that doesn't exist, does the script fail? The other version of this PR checked that they were already installed IIRC.

Reply:
As I wrote this I thought it would probably fail and that's OK, but I'm not certain.

An error like "Error: Cask 'google-chrome' is unavailable: No Cask with this name exists" would be printed, even in https://github.com/web-platform-tests/wpt/pull/21099, but whether that fails the step depends on how Azure Pipelines wraps multi-line scripts, i.e. whether it has the equivalent of `set -x` somewhere.

I'll merge this to unblock https://github.com/web-platform-tests/wpt/pull/21097 and the experiment in another PR.

Reply:
It doesn't fail, see https://github.com/web-platform-tests/wpt/pull/21113#issuecomment-572643705.

Appending `|| true` would make this more explicit.